shield.go

```diff
@@ -17,6 +18,8 @@ const MaxLength = 4096
 // SaltLength is the length of the salt generated by GenerateSalt.
 const SaltLength = 32
 
+var ErrPasswordTooLong = errors.New("password too long")
```

Comment:
Actually, you might be right, see [constant erros](http://dave.cheney.net/2016/04/07/constant-errors) and the referenced article.